mythical-creatures/test/hobbit-test.js

```diff
@@ -37,8 +37,8 @@ describe('Hobbit', function () {
     assert.equal(hobbit.age, 3);
   });
 
-  function timeTravel(num, hobbit) {
-    for(var i = 0; i < num; i++) {
+   var timeTravel = (num, hobbit) => {
+    for (var i = 0; i < num; i++) {
```

Comment:
Do we want to keep this as a regular function since this is Hobbit?  And introduce es6 arrow functions in student made ones?